juntagrico/templates/iso20022/share_pain.001.xml

```diff
@@ -63,7 +63,7 @@
                     </Id>
                 </CdtrAcct>
                 <RmtInf>
-                    <Ustrd>{% blocktrans %}{{ v_share }} Rückzahlung{% endblocktrans %}</Ustrd>
+                    <Ustrd>{% blocktrans with sn=share.number sid=share.id %}{{ v_share }} Rückzahlung | {{ sn }}(Intern: {{ sid }} ){% endblocktrans %}</Ustrd>
```

Comment:
Also if share id is equal share number, it should only be displayed once.

Reply:
since the number is only there to represenet a world before the share had ids they are probably never the same number, so i replaced the none by an empty string and that should suffice